core/drivers/pm/sam/at91_pm.c

```diff
@@ -236,6 +236,7 @@ static int at91_pm_config_ws_ulp1(bool set)
  */
 static bool at91_pm_verify_clocks(void)
 {
+	int pck_cnt = 4;
```

Comment:
Please add and use an `AT91_PMC_PKC_COUNT` define depending on `CFG_SAMA7G5` instead of using magic numbers here.